### PR TITLE
[service.subtitles.thaisubtitle] 1.1

### PR DESCRIPTION
**Pull request by djay**

### CODE REVIEW

File: service.subtitles.thaisubtitle/service.py [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+import urllib
+import shutil
+import unicodedata
+import re
+import string
+import difflib
+import HTMLParser
+from thaisubtitles import getallsubs, search_manual, search_movie
+
+__addon__ = xbmcaddon.Addon()
```
Comment (learningit) [inline service.subtitles.thaisubtitle/service.py]: In your next update please remove \__name\__ style labels, these are for python internals.

**Comment by learningit** [comment]
Please remove the .gitignore from the commit. Once that's done, I'll merge.

**Comment by djay** [comment]
What's wrong with __name__ ? It allows the search function to be tested
from the command line and has no effect on the running of the plugin. It is
a useful debug feature.

**Comment by learningit** [comment]
I didn't mean \__name\__ literally, the style of using \__ before and after labels is highly discouraged as they are meant to be used only for reserved/magic values internal to python. It's bad coding practice and confusing to someone reading the code. Accidentally choosing the wrong label could have strange side effects as well.

File: service.subtitles.thaisubtitle/.gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.idea/
```
Comment (phil65) [inline service.subtitles.thaisubtitle/.gitignore]: please remove this file from the PR.

**Comment by phil65** [comment]
Since this is a new add-on addition the PR should point to the "jarvis" branch.

**Comment by djay** [comment]
@phil65 The documentation said I can point to whichever version I thought was compatible. Wouldn't pointing to jarvis mean my addon won't be installed for older versions?

**Comment by djay** [comment]
@learningit I'm not sure what you mean magical. What I'm using is standard python and is in the python documentation - https://docs.python.org/3/library/__main__.html
If it makes you uncomfortable I will move the code to another module called run.py

**Comment by phil65** [comment]
@djay we dont support older kodi versions for newly added add-ons.

**Comment by phil65** [comment]
Closing this, new add-ons have to be jarvis+

**Pull request closed without merging**